### PR TITLE
fix: standardize Docker containers and containerized start-up

### LINKED ISSUE #31: Cleanup types
As a developer, we needs types and tightly bound typescript to ensure that there are no errors in the code.

# Problem:
- We are using `any` in a few places where we should have a type.
- We have types that may not be correct.
- The type files are hard to follow.

# Solution:
- Remove as many `any`s as possible
- confirm all types are correct
- define any undefined types
- cleanup file names and contents where possible

### PR DESCRIPTION
**Pull request by JoeCap08055**
* Update the building of Docker containers per our standards [here](https://docs.google.com/document/d/1Iv5RDx6-OV4S0vnn8DG4b6-zia0qkQpXa0Y9m7ihG5Y/edit#heading=h.s8rj6z75559l)
* Update launching of services for local testing in `docker-compose.yaml`
* Update launching instructions in README

Closes #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,38 +18,46 @@ For example, here are some interactions that are provided by account-service:
...
+### 1. Clone the Repository
 
-   ```bash
-   docker-compose -f docker-compose.dev.yaml --profile instant up 
-   ```
+Clone the Content Publisher repository to your local machine:
 
-   This will start Frequency, Redis and Account Service api/worker containers.
+```bash
+git clone https://github.com/amplicalabls/account-service.git
```
Comment (mattheworris) [inline README.md:35]:
```suggestion
git clone https://github.com/AmplicaLabs/account-service.git
```

File: .env.docker.dev [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copy this file to ".env.dev" and ".env.docker.dev", and then tweak values for local development
```
Comment (mattheworris) [inline .env.docker.dev:1]: Question: Do we need this file in the repo? I understood that it was to be copied from env.template.

File: README.md [hunk 3]
```diff
@@ -113,7 +121,7 @@ In order to run the account-service in development mode without containers, you
 3. Set breakpoints in the code and debug your code.
 
 4. Monitor the service worker jobs in [BullUI](http://0.0.0.0:3000/queues/).
-  
+
    Any API functions that require an extrinsic to submitted to the blockchain will be queued here. The queue will manage the amount of `capacity` this service is allowed to use.
```
Comment (mattheworris) [inline README.md:125]:
```suggestion
   Any API functions that require an extrinsic to be submitted to the blockchain will be queued here. The queue will manage the amount of `capacity` this service is allowed to use.
```

File: .env.docker.dev (hunk 2 above)
Comment (JoeCap08055) [reply .env.docker.dev:1]: Following the pattern of the other repos, most have it committed. I somewhat agree, but I think we need further review & agreement on how this should work. Maybe we should provide multiple `.env.*.template` files, then, for different scenarios. Let's consider it out of scope to change the pattern in this ticket.

File: docker-compose.yaml [hunk 4]
```diff
@@ -8,88 +6,77 @@ services:
...
+    # built by Frequency at the moment, and auto-pull won't work otherwise
     platform: linux/amd64
+    # Uncomment SEALING_MODE and SEALING_INTERVAL if you want to use interval sealing.
+    # Other options you may want to add depending on your test scenario.
+    # environment:
+    #   - SEALING_MODE=interval
+    #   - SEALING_INTERVAL=3
+    #   - CREATE_EMPTY_BLOCKS=true
+    # Uncomment below if you want to let the chain run and keep all of the historical blocks
+    # command: --state-pruning=archive
```
Comment (mattheworris) [inline docker-compose.yaml:23]: Question: Can we also support starting with the offchain indexer here now?
account-service requires the offchain indexer for one of the endpoints.

Comment (JoeCap08055) [reply docker-compose.yaml:23]: Yes, you can pass any Frequency CLI parameters in the `command`

**Review by mattheworris — APPROVED** [review]
- Reviewed code changes

🚢 it!
